ruma-events/Cargo.toml

```diff
@@ -27,7 +27,14 @@ maplit = "1.0.2"
 matches = "0.1.8"
 ruma-identifiers = { version = "0.16.2", path = "../ruma-identifiers", features = ["rand"] }
 trybuild = "1.0.30"
-criterion = "0.3.2"
+
+[dependencies.criterion]
+version = "0.3.2"
+optional = true
+
+[features]
+default = []
+criterion_bench = ["criterion"]
```

Comment:
Oh, by the way you don't need this. Every optional crate is its own feature. That is, you can just leave this out and change the instructions from `--feature criterion_bench` to `--feature criterion`.

Reply:
Cool, learning all sorts of useful stuff today!